cache-ehcache/build.gradle

```diff
@@ -15,6 +15,6 @@ dependencies {
 
 micronautBuild {
     resolutionStrategy {
-        force "org.ehcache:ehcache-clustered:$ehcacheVersion"
+        force "org.ehcache:ehcache-clustered:${libs.versions.managed.ehcache.get()}"
```

Comment:
Perhaps we can get rid of this. I suppose it was introduced to avoid dependabot trying to upgrade it

/cc @melix

Reply:
I'll do that as part of the ehcache upgrade here https://github.com/micronaut-projects/micronaut-cache/issues/523